src/status_im/contexts/chat/messenger/composer/actions/view.cljs

```diff
@@ -43,8 +43,12 @@
 
 (defn f-send-button
   [props state animations window-height images? btn-opacity z-index edit]
-  (let [{:keys [text-value]} state
-        customization-color  (rf/sub [:profile/customization-color])]
+  (let [{:keys [text-value]}              state
+        customization-color               (rf/sub [:profile/customization-color])
+        {:keys [color chat-id chat-type]} (rf/sub [:chats/current-chat-chat-view])
+        {contact-customization-color
+         :customization-color}            (when (= chat-type constants/one-to-one-chat-type)
+                                            (rf/sub [:contacts/contact-by-address chat-id]))]
```

Comment:
seems like we're using this sub a few times to only get the contact's customization color. Might be better to have a separate sub e.g. `:contacts/customization-color-by-address` WDYT?